arbnode/mel/state.go

```diff
@@ -90,12 +90,12 @@ func (s *State) Clone() *State {
 	}
 }
 
-func (s *State) AccumulateMessage(msg *arbostypes.MessageWithMetadata) *State {
+func (s *State) AccumulateMessage(msg *arbostypes.MessageWithMetadata) error {
 	// TODO: Unimplemented.
```

Comment:
Should they return unimplemented errors instead of nils?

Reply:
The tests would fail, as this is part of the message extraction function